### PR TITLE
Extend readme

### PR DESCRIPTION
**Pull request by eszthoff**

### CODE REVIEW

File: example/src/components/ButtonContainer/ButtonContainer.jsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from 'react';
 import './ButtonContainer.css'
 
+/**
+ * One custom component from a application that uses Nice!, like Search!
```
Comment (carlobernardini) [inline example/src/components/ButtonContainer/ButtonContainer.jsx]: I'm not sure what this means

File: README.md [hunk 2]
```diff
@@ -1,17 +1,59 @@
-Nice 2
-======
+# Nice 2
 
 [![Build Status](https://travis-ci.com/textkernel/nice2.svg?branch=master)](https://travis-ci.com/textkernel/nice2)
 
 Rract Nice UI kit with theme support
```
Comment (carlobernardini) [inline README.md]: Rrrrrract

Comment (eszthoff) [reply README.md]: gggrrrr

**Comment by carlobernardini** [comment]
@eszthoff LGTM

File: example/src/components/ButtonContainer/ButtonContainer.jsx (hunk 1 above)
Comment (eszthoff) [reply example/src/components/ButtonContainer/ButtonContainer.jsx]: I agree. But that component might change in `creating-example-app` branch, so should add a comment about it there.

**Pull request merged**